packages/openapi-generator/src/parser/operation.ts

```diff
@@ -38,20 +38,21 @@ export function parseOperation(
 /**
  * Parse parameters of an operation.
  * @param operation The original operation definition.
- * @param refs List of crossreferences that can occur in the document.
+ * @param refs List of cross references that can occur in the document.
  * @returns A list of parsed parameters.
  */
 export function parseParameters(
   operation: OpenAPIV3.OperationObject,
   refs: $Refs
 ): OpenApiParameter[] {
   // TODO: What if this is a reference? What does OpenApi do?
-  // TODO: What about oneof and other operations?
+  // TODO: What about one of and other operations?
   return (
     operation.parameters
       ?.map(param => resolveObject(param, refs))
       .map(param => ({
         ...param,
+        name: camelCase(param.name),
```

Comment:
Is this the point where the `$` and `.` are removed?

Reply:
Yes, the function `camelCase` removes the special characters and format it to camel case.